packages/components/charts/package.json

```diff
@@ -46,6 +46,7 @@
     "gen:docs": "tsx ../../../scripts/generate-docs"
   },
   "dependencies": {
+    "@types/lodash": "^4.17.7",
```

Comment:
Please move `devDependencies`.

Reply:
Doing so will result in a type error in `use-radial-chart.ts`.

Reply:
If you use `ComponentPropsWithoutRef` instead of `ComponentPropsWithRef`, you do not need `"@types/lodash"` itself.